foundry.toml

```diff
@@ -22,7 +22,7 @@
   mainnet = { key = "${API_KEY_ETHERSCAN}" }
   optimism = { key = "${API_KEY_OPTIMISTIC_ETHERSCAN}" }
   polygon = { key = "${API_KEY_POLYGONSCAN}" }
-  mumbai = { key = "${API_KEY_POLYGONSCAN}" }
+  amoy = { key = "${API_KEY_POLYGONSCAN}" }
```

Comment:
Umm... I don't think this is going to work.😅

Reply:
I have pushed a fix in https://github.com/Open-Attestation/document-store/pull/168/commits/5a56e54df446683a557b01637a758119dd6c3172 for this.

Other than this, looks good!👍

Reply:
Oops, thanks for helping with this part!